test/libdnf/rpm/test_package.cpp

```diff
@@ -0,0 +1,346 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "test_package.hpp"
+
+#include "../utils.hpp"
+
+#include "libdnf/rpm/solv_query.hpp"
+
+#include <vector>
+
+
+CPPUNIT_TEST_SUITE_REGISTRATION(RpmPackageTest);
+
+
+libdnf::rpm::Package RpmPackageTest::get_pkg(const std::string &nevra) {
+    libdnf::rpm::SolvQuery query(sack.get());
+    query.ifilter_nevra(libdnf::sack::QueryCmp::EQ, {nevra});
+    CPPUNIT_ASSERT_MESSAGE("get_pkg(\"" + nevra + "\"): no package found.", query.size() > 0);
+    return *query.get_package_set().begin();
+}
+
+
+void RpmPackageTest::test_equality() {
+    auto package1 = get_pkg("CQRlib-1.1.1-4.fc29.x86_64");
+    auto package2 = get_pkg("CQRlib-1.1.1-4.fc29.x86_64");
+    CPPUNIT_ASSERT(package1 == package2);
+
+    auto package3 = get_pkg("nodejs-1:5.12.1-1.fc29.x86_64");
+    CPPUNIT_ASSERT(package1 != package3);
+}
+
+
+void RpmPackageTest::test_get_id() {
+    get_pkg("CQRlib-1.1.1-4.fc29.x86_64").get_id();  // Can't really test for the value of the ID
```

Comment:
In future we can load particular rpm in certain order - like packages from commandline. Then we will know ids.
For the present time we can make a test that `get_pkg("CQRlib-1.1.1-4.fc29.x86_64").get_id() != 0`.